pkg/prometheus/promcfg_test.go

```diff
@@ -1244,6 +1244,49 @@ func TestAlertmanagerEnableHttp2(t *testing.T) {
 	}
 }
 
+func TestAlertmanagerAlertRelabelConfigs(t *testing.T) {
```

Comment:
We need sub-tests depending on the Prometheus version.

Reply:
Could you elaborate, or point me to an example?